curvefs/src/mds/topology/topology_manager.cpp

```diff
@@ -871,12 +852,20 @@ void TopologyManager::ListPartition(const ListPartitionRequest *request,
         info->set_start(partition.GetIdStart());
         info->set_end(partition.GetIdEnd());
         info->set_txid(partition.GetTxId());
+        info->set_full(partition.GetFull());
+        if (partition.GetInodeNum() != UNINTIALIZE_ID) {
```

Comment:
compare number of inode with an ID is misleading

Reply:
done